main.js

```diff
@@ -160,4 +163,115 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+//opdracht 1a
+console.log("opdracht 1a");
+const output = inventory.map(tv => tv.name);
+console.log(output);
+
+
+//opdracht 1b
+console.log("opdracht 1b");
+inventory.map((tv) => {
+  if (tv.sold >= tv.originalStock){
+    console.log(tv);
+  }
+});
+//opdracht 1c
+console.log("opdracht 1c");
+const ambiLight = inventory.find((tv) => {
+  if (tv.options.ambiLight === true){
```

Comment:
`boolean_waarde === true` is altijd gelijk aan `boolean_waarde`, dus in dit geval
```suggestion
  if (tv.options.ambiLight){
```